src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -54,30 +49,23 @@ export const IncidentMap = () => {
     },
     [setMapMessage, setShowMessage]
   )
+  /* istanbul ignore next */
+  const handleIncidentSelect = useCallback((incident) => {
+    setSelectedIncident(incident)
+    incident && setDrawerState(DrawerState.Open)
```

Comment:
huh, first setSelectedIncident and then check incident for not undefined?

Reply:
Legacy of another try...